cypress/integration/transfer/disabled-transfer-buttons.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Disable transfer buttons when actions are not possible
+
+  Scenario: No highlighted items in options list
+    Given an options list that has items
+    And none of the items in the options list are highlighted
+    When the 'move to selected list' button is pressed
```

Comment:
Actually, now that I realize that the "move up" button is disabled, I'm wondering why the "move to selected list" button is not disabled. Same for the other scenarios in this feature file (which is actually called "disabled transfer buttons"

Reply:
I prematurely removed those scenario when I thought it was wrong to reference buttons being disabled 😅 I've added these back in now.